backend/src/app/config.clj

```diff
@@ -48,6 +48,7 @@
    :database-uri "postgresql://postgres/penpot"
    :database-username "penpot"
    :database-password "penpot"
+   :authenticated-cookie-domain "penpot.app"
```

Comment:
I think we should not have it defaults, this feature is mainly "internal"

Reply:
This is a provisonal commit I made for local testing, will be removed prior to merge.